src/dash-table/components/Table/props.ts

```diff
@@ -43,6 +60,7 @@ export type PaginationMode = 'fe' | 'be' | boolean;
 export type RowSelection = 'single' | 'multi' | false;
 export type SelectedCells = CellCoordinates[];
 export type SetProps = (...args: any[]) => void;
+export type SetState = (state: Partial<IState>) => void;
```

Comment:
The controlled table can only modify the basic state, not the extended "standalone" state